jekyll-exporter.php

```diff
@@ -125,7 +250,7 @@ function get_posts() {
 		}
 
 		$posts      = array();
-		$post_types = apply_filters( 'jekyll_export_post_types', array( 'post', 'page', 'revision' ) );
+		$post_types = apply_filters( 'jekyll_export_post_types', array( 'post', 'page' /*, 'revision'*/ ) );
```

Comment:
?

Reply:
Oops forgot to remove that. Converted as options and reverted to the default.

On my WordPress database, exporting revisions was doing strange things and some revisions were exporting with other posts metadata... I did not figure out why, my WordPress database seemed OK and separate functions calls also... Maybe another strange PHP or WordPress behavior that made me comment the global $post.  But I am not skilled enough in PHP or WordPress to debug further.